src/main/java/org/cascadebot/cascadebot/messaging/MessagingUI.java

```diff
@@ -176,7 +176,7 @@ public void replyUsage() {
 
     public void replyUsage(ExecutableCommand command) {
         String usage = context.getUsage(command);
-        List<Page> pages = PageUtils.splitStringToEmbedPages(usage, context.i18n("commands.usage.title", command.command()), 1000, '\n');
+        List<Page> pages = PageUtils.splitStringToEmbedPages(usage, context.i18n("commands.usage.title", command.command()), 1500, '\n');
```

Comment:
Another arbitrary number

Reply:
This can be reverted, was just me messing about